Library/Homebrew/os/mac/xcode.rb

```diff
@@ -76,6 +76,8 @@ def uncached_version
 
         return "0" unless OS.mac?
 
+        return nil unless MacOS::Xcode.installed? && MacOS::CLT.installed?
```

Comment:
Mind not using `&&` with `unless`? Maybe just me being stupid but I always find it hard to parse.